app/services/AtsService.scala

```diff
@@ -134,7 +135,7 @@ class AtsService @Inject()(
             "time"      -> new Date().toString
           )
         )
-      case _: SaUtr =>
+      case Some(_: SaUtr) =>
```

Comment:
What happens on None?

Reply:
This cannot be None. account comes from     val account = utils.AccountUtils.getAccount(request)
 which does not return a None

Reply:
You need to handle the None case or you will get a pattern match exception